kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/DefaultKroxyliciousTester.java

```diff
@@ -147,9 +155,20 @@ public <U, V> Consumer<U, V> consumer(String virtualCluster, Serde<U> keySerde,
         return clients(virtualCluster).consumer(keySerde, valueSerde, additionalConfig);
     }
 
+    public void restartProxy() {
+        try {
+            proxy.close();
+            proxy = new KafkaProxy(kroxyliciousConfig).startup();
+        }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
     @Override
     public void close() {
         try {
+            clients.values().forEach(KroxyliciousClients::close);
```

Comment:
Similar to the comment in KroxyliciousClients, we want to try to close `proxy` and all instances of KroxyliciousClient, even if one KroxyliciousClients::close throws.